soh/src/overlays/actors/ovl_Bg_Spot06_Objects/z_bg_spot06_objects.c

```diff
@@ -30,6 +30,8 @@ typedef enum {
 #define WATER_LEVEL_LOWERED (WATER_LEVEL_RAISED - 680)
 #define WATER_LEVEL_RIVER_LOWERED (WATER_LEVEL_RIVER_RAISED - 80)
 
+#define WATER_LEVEL_RIVER_LOWER_Z 2203
```

Comment:
bad merge commit? lol

Reply:
It's comparing an older branch to develop. The shown changes don't matter as this just exists to provide download links to a build.